src/Components/AlbumCardsDisplay/AlbumCardsDisplay.css

```diff
@@ -0,0 +1,12 @@
+.card-container {
+  background-color: #CDD6DD;
+  display: flex;
+
+  justify-content: center;
+  align-content: center;
+  width: 100vw;
```

Comment:
The width of the container might change slightly now that I've defined css grid on the landing page, 100% might be more representative of the wireframes